src/rai/rai/utils/configurator.py

```
@@ -0,0 +1,528 @@
+        elif embeddings_model_vendor_name == "ollama":
+            embeddings_model = OllamaEmbeddings(
+                model=st.session_state.config["ollama"]["embeddings_model"],
+                base_url=st.session_state.config["ollama"]["base_url"],
+            )
+
+        progress.progress(0.4)
+        use_langfuse = st.session_state.config["tracing"]["langfuse"]["use_langfuse"]
+        if use_langfuse:
+            if not os.getenv("LANGFUSE_SECRET_KEY", "") or not os.getenv(
+                "LANGFUSE_PUBLIC_KEY", ""
+            ):
```

Ruff

Use a single `if` statement instead of nested `if` statements

(SIM102)